app/controllers/invitation_controller.rb

```diff
@@ -9,8 +9,11 @@ def new
 
   def create
     user = User.find_by_name(params[:user][:name].strip)
+
     team = AssignmentTeam.find(params[:team_id])
     student = AssignmentParticipant.find(params[:student_id])
+    student_email = student.user_id
```

Comment:
What?  User_ID normally doesn't contain the domain, but email needs to.  Sounds like a bad variable name, or worse.